src/ar/ARTester.cpp

```diff
@@ -14,25 +14,33 @@
 #include "../camera/CameraParams.h"
 #include "Detector.h"
 
+constexpr int NUM_SKIP = 5;
+
 const std::string WINDOW_NAME = "Image";
 
 const std::string keys =
 	"{h help             |        | Show this help message.}"
 	"{cam_config c       | <none> | The path to a camera configuration file "
 	"defining the camera you would like to use.}"
-	"{cam_override co    | -1     | If present, should be the ID of a camera to open "
+	"{file_override fo   |        | If present, should be the name of a video file to open "
 	"rather than the one defined in the configuration file.}"
+	"{cam_override co    | -1     | If present, should be the ID of a camera to open "
+	"rather than the one defined in the configuration file. NOTE: Will take precedence over "
+	"file_override if both are present.}"
 	"{marker_set m       | urc    | The set of markers to look for. Currently "
 	"only \"urc\" and \"circ\" are supported.}"
 	"{frame_by_frame f   |        | If present, program will go frame-by-frame "
 	"instead of capturing continuously.}";
 
-cam::Camera cap;
```

Comment:
Yeah, in particular it would be good to be able to switch between video input vs. webcam input